lisa/util/process.py

```diff
@@ -77,7 +77,7 @@ def start(
             if not self._is_posix:
                 split_command = ["cmd", "/c", command]
             elif sudo:
-                command = f"sudo -s {command}"
+                command = f"sudo -s sh -c '{command}'"
```

Comment:
Try to construct a string list like below, not rely on shlex.
`split_command = ["sudo", "sh", "-c", command]`